framework/core/src/Extension/Extension.php

```diff
@@ -498,7 +496,7 @@ public function migrate(Migrator $migrator, $direction = 'up')
         }
 
         if ($direction == 'up') {
-            return $migrator->run($this->getPath().'/migrations', $this);
+            $migrator->run($this->getPath().'/migrations', $this);
```

Comment:
Shouldn't this also be a return? If the migrator run fails, feels like we should return that as well.

Reply:
the `run` method of the migrator doesn't actually return anything, same goes for methods called within `run`.

Reply:
Fair enough!